plugins/PostProcessingPlugin/scripts/RetractContinue.py

```diff
@@ -27,58 +30,90 @@ def getSettingDataString(self):
             }
         }"""
 
-    def execute(self, data):
-        current_e = 0
-        current_x = 0
-        current_y = 0
-        current_z = 0
+    def _getTravelMove(self, travel_move: str, default_pos: Vector) -> Tuple[Vector, float]:
+        travel = Vector(
+            self.getValue(travel_move, "X", default_pos.x),
+            self.getValue(travel_move, "Y", default_pos.y),
+            self.getValue(travel_move, "Z", default_pos.z)
+        )
+        f = self.getValue(travel_move, "F", -1.0)
+        return travel, f
+
+    def _travelMoveString(self, travel: Vector, f: float, out_e: float) -> str:
```

Comment:
Nitpicking, "out_e" might need a bit of documentation. It wasn't immediately clear to me what it meant.

Reply:
fixed by renaming parameter.